tests/testUtil.py

```diff
@@ -55,10 +60,16 @@ def countCDXJEntries(cdxjData):
 
 def startReplay(warcFilename):
     global p
-    pathOfWARC = os.path.join(os.path.dirname(__file__) +
-                              '/../samples/warcs/' + warcFilename)
-    tempFilePath = tempfile.gettempdir() + '/' + ''.join(random.sample(
-        string.ascii_uppercase + string.digits * 6, 12)) + '.cdxj'
+    pathOfWARC = os.path.join(
+        str(Path(os.path.dirname(__file__)).parents[0]) +
+        os.path.sep.join(['', 'samples', 'warcs', warcFilename]))
+
+    number_of_characters = 12
+    character_set = string.ascii_uppercase + string.digits * 6
+
+    tempFilePath = (
+        f"{tempfile.gettempdir()}{os.path.sep}"
+        f"{random.sample(character_set, number_of_characters)}.cdxj")
 
     open(tempFilePath, 'a').close()  # Create placeholder file for replay
```

Comment:
```suggestion
    tf = tempfile.NamedTemporaryFile(mode='a', suffix='.cdxj')
    tempFilePath = tf.name
    tf.close()
```

Reply:
Great adaptation to use the native libs. Getting less hacky with each PR. :-)